test/IISIntegration.FunctionalTests/Utilities/IISFunctionalTestBase.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Xunit.Abstractions;
+
+namespace IISIntegration.FunctionalTests.Utilities
+{
+    public class IISFunctionalTestBase : FunctionalTestsBase
```

Comment:
Still need to figure out how this applies to mixed server scenarios. Let's talk.